### DISCUSSION TITLE
Finding "dam failure" parameter for flash flood warnings

### DISCUSSION THREAD

**Discussion by mbg206** [weather-gov/api#736] [Q&A]
Is there a way to find the value of the "DAM FAILURE" parameter from the `api.weather.gov/alerts` endpoint? I noticed this alert recently issued from weather.gov:

```
Flash Flood Warning
MNC013-242130-
/O.NEW.KMPX.FF.W.0008.240624T1534Z-240624T2130Z/
/00000.U.DM.000000T0000Z.000000T0000Z.000000T0000Z.OO/

BULLETIN - EAS ACTIVATION REQUESTED
Flash Flood Warning
National Weather Service Twin Cities/Chanhassen MN
1034 AM CDT Mon Jun 24 2024

The National Weather Service in the Twin Cities has issued a

* Flash Flood Warning for...
  A Dam Break on the Blue Earth River below Rapidan Dam in...
  Central Blue Earth County in south central Minnesota...

* Until 430 PM CDT.

* At 1034 AM CDT, dam operators reported the river has eroded earth
  around the dam and is uncontrolled. This may result in flash
  flooding downstream along the Blue Earth River.

  HAZARD...Flash flooding from the failure of Rapidan Dam.

  SOURCE...Dam operator.

  IMPACT...Flooding of areas downstream from the Rapidan Dam
           along the Blue Earth.

This includes the following streams and drainages...
  Minneopa Creek, Blue Earth River and Le Sueur River.
Flooding is occurring along the Blue Earth.

PRECAUTIONARY/PREPAREDNESS ACTIONS...

If you are in low lying areas below the Rapidan Dam you should move
to higher ground immediately.

&&

LAT...LON 4412 9413 4414 9411 4414 9409 4414 9407
      4413 9405 4412 9404 4412 9405 4412 9406
      4412 9407 4412 9408 4411 9410 4410 9411
      4409 9410 4409 9412 4409 9413 4410 9414

FLASH FLOOD...OBSERVED
DAM FAILURE...OCCURRING

$$

Borghoff
```

From the parameters listed from the [API](https://api.weather.gov/alerts/urn:oid:2.49.0.1.840.0.c43e8f56e6dc3890ab9c41c469f59945f8869624.001.1), there is only `flashFloodDetection` (set to `"OBSERVED"`) but nothing related to a dam failure, although it seems to be listed in the text version of the alert pulled from weather.gov. Is there any reason for this?

**Comment by dwhitemv25** [comment]
[NWS CAP specification documentation](https://vlab.noaa.gov/web/nws-common-alerting-protocol/cap-documentation).

I don't see a parameter defined for the flood cause in the NWS CAP spec. The narrative text is written by the forecaster, but there is machine-readable encoding in the two VTEC lines (bracketed with slashes `/.../`). [VTEC documentation](https://www.weather.gov/vtec/).
